app/src/main/java/org/systers/mentorship/models/Relationship.kt

```diff
@@ -22,7 +22,7 @@ import kotlinx.android.parcel.Parcelize
  * @param state state of the relation (@link to RelationState)
  * @param notes notes related to the mentorship relation
  */
-@SuppressLint("ParcelCreator")
```

Comment:
Since you're removing this line you should remove the import for SuppressLint too